package.json

```diff
@@ -40,8 +40,10 @@
     "node": ">=18"
   },
   "dependencies": {
+    "@nodesecure/github": "^1.2.0",
     "cacache": "^18.0.0",
     "deepmerge": "^4.3.1",
+    "kleur": "^4.1.5",
```

Comment:
devDependencies

Reply:
We should even remove `kleur`, useless devDep imo
See [this example](https://github.com/TopCli/prompts/pull/119)